src/Administration/Resources/app/administration/build/plugins.vite.ts

```diff
@@ -186,13 +187,13 @@ const main = async () => {
             }
 
             if (extension.isApp) {
-                swPluginDevJsonData[extension.technicalName].html = `http://localhost:${availablePorts[index]}/index.html`;
+                swPluginDevJsonData[extension.technicalName].html = `http://${host}:${availablePorts[index]}/index.html`;
             }
 
             if (extension.isPlugin) {
-                swPluginDevJsonData[extension.technicalName].js = `http://localhost:${availablePorts[index]}/${fileName}`;
+                swPluginDevJsonData[extension.technicalName].js = `http://${host}:${availablePorts[index]}/${fileName}`;
                 swPluginDevJsonData[extension.technicalName].hmrSrc =
-                    `http://localhost:${availablePorts[index]}/@vite/client`;
+                    `http://${host}:${availablePorts[index]}/@vite/client`;
```

Comment:
it would be nice we could expose just one port, we do that on storefront watcher too, so less port mapping